org.mixedrealitytoolkit.uxcore/Button/PressableButton.cs

```diff
@@ -474,7 +525,14 @@ protected override void OnHoverExited(HoverExitEventArgs args)
             {
                 // Remove from our valid poke hash set if it was registered there.
                 validPokeInteractors.Remove(pokeInteractor);
+
+                if (validPokeInteractors.Count == 0)
+                {
+                    isHovered = false;
```

Comment:
We should be reusing the base `IsHovered` value

Reply:
I missed that, thanks for catching it and letting me know.  I removed it in the latest commit.  Resolving this.